src/OTAHandlers.cpp

```diff
@@ -35,8 +40,6 @@ void OTAHandlers::initOTA(uint8 ep)
     #if TRACE_OTA_DEBUG
         vDumpCurrentImageOTAHeader(otaEp);
     #endif //TRACE_OTA_DEBUG
-
-    vDumpOverridenMacAddress(); // Dump MAC address anyway to avoid au8MacAddress to be optimized out
```

Comment:
Please move this call little up into #if TRACE_OTA_DEBUG section.

Reply:
done